src/goddard/renderer.c

```diff
@@ -164,25 +164,25 @@ static s16 *sPickBuf;                                // @ 801BE788
 static LookAt D_801BE790[2];
 static LookAt D_801BE7D0[3];
 #if defined(VERSION_JP) || defined(VERSION_US)
-static OSMesgQueue D_801BE830; // controller msg queue
-static OSMesg D_801BE848[10];
+static OSMesgQueue sGdContMesgQueue; // controller msg queue
+static OSMesg sGdContMesgArray[10];
 UNUSED static u32 unref_801be870[16];
-UNUSED static OSMesgQueue D_801BE8B0;
+UNUSED static OSMesgQueue sGdDMACompleteMsgQueue;
 static OSMesgQueue sGdDMAQueue; // @ 801BE8C8
 UNUSED static u32 unref_801be8e0[25];
 static OSMesg sGdMesgBuf[1]; // @ 801BE944
 UNUSED static u32 unref_801be948[13];
-static OSMesg sGdDMACompleteMsg; // msg buf for D_801BE8B0 queue
+static OSMesg sGdDMACompleteMsg; // msg buf for sGdDMACompleteMsgQueue queue
 static OSIoMesg sGdDMAReqMesg;
-static struct ObjView *D_801BE994; // store if View flag 0x40 set
+static struct ObjView *stored_view; // store if View flag 0x40 set
 #endif
 
 // data
 UNUSED static u32 unref_801a8670 = 0;
 static s32 D_801A8674 = 0;
 UNUSED static u32 unref_801a8678 = 0;
-static s32 D_801A867C = 0;
-static s32 D_801A8680 = 0;
+static s32 D_801A867C = 0; // useless
```

Comment:
Probably worth marking the 'useless' variables with `UNUSED`. Or better yet, these could probably be removed completely if there's no memory sorcery surrounding their existence.